src/pip/_internal/commands/install.py

```diff
@@ -191,6 +191,7 @@ def __init__(self, *args, **kw):
         cmd_opts.add_option(cmdoptions.no_binary())
         cmd_opts.add_option(cmdoptions.only_binary())
         cmd_opts.add_option(cmdoptions.prefer_binary())
+        cmd_opts.add_option(cmdoptions.prefer_local_compatible())
```

Comment:
Wait - on the original issue, I made the point very strongly that I was against this being added to `pip install`.